research/sentiment_analysis/data/imdb.py

```diff
@@ -1,46 +1,47 @@
-from data.util import pad_sentence, to_dataset, START_CHAR, OOV_CHAR
-import tensorflow as tf, numpy as np
+from data.util import OOV_CHAR
+from data.util import pad_sentence
+from data.util import START_CHAR
+import numpy as np
+import tensorflow as tf
 
 NUM_CLASS = 2
 
 
-def construct_input_fns(vocabulary_size, sentence_length,
-                        batch_size, repeat=1):
-  """Returns training and evaluation input functions.
+def load(vocabulary_size, sentence_length):
+  """Returns training and evaluation input for imdb dataset.
 
   Args:
     vocabulary_size: The number of the most frequent tokens
       to be used from the corpus.
     sentence_length: The number of words in each sentence.
       Longer sentences get cut, shorter ones padded.
-    batch_size: Number of data in each batch.
-    repeat: The number of epoch.
   Raises:
     ValueError: if the dataset value is not valid.
   Returns:
-    A tuple of training and evaluation input function.
+    A tuple of length 4, for training and evaluation data,
+    each being an numpy array.
   """
   (x_train, y_train), (x_test, y_test) = tf.keras.datasets.imdb.load_data(
       path="imdb.npz",
       num_words=vocabulary_size,
       skip_top=0,
       maxlen=None,
-      seed=113,
+      seed=112,
```

Comment:
Why did this change?

Reply:
It was likely to be changed while I was adjusting the hyper parameter. I am redoing the run to see the same accuracy is achieved with 113 seed.

Reply:
I verified that the run with the seed of 113 gives 90.5% accuracy.